python/app/syntax_highlighter.py

```diff
@@ -9,23 +9,9 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 import keyword as py_keywords
+import builtins
 
-# NOTE: This repo is typically used as a Toolkit app, but it is also possible use the console in a
```

Comment:
I think we can leave this comment, it seems informative.

Reply:
This comment seems to explain the try/except block that I modified.

Reply:
Can you leave part of the comment please?


```python
# NOTE: This repo is typically used as a Toolkit app, but it is also possible use the console in a
# standalone fashion.
# Flame, for example, uses the console when there is no Toolkit engine running.
```